test/models/licence-version.model.test.js

```diff
@@ -64,5 +67,39 @@ describe('Licence Version model', () => {
         expect(result.licence).to.equal(testLicence)
       })
     })
+
+    describe('when linking to licence version purposes', () => {
```

Comment:
I've taken the example set out in `user.model.test.js`

```suggestion
    describe('when linking through licence version purposes to purposes', () => {
```